contracts/market/template/MarketExtended.sol

```diff
@@ -45,8 +45,8 @@ contract MarketExtendedCore is AccessControlledAndUpgradeableModifiers, MarketSt
 
     (uint80 latestRoundId, int256 initialAssetPrice, , , ) = oracleManager.chainlinkOracle().latestRoundData();
     epochInfo.latestExecutedOracleRoundId = latestRoundId;
-    require(uint256(initialAssetPrice) < type(uint128).max); // Be double sure the chainlink price isn't broken
-    epochInfo.lastEpochPrice = uint128(uint256(initialAssetPrice));
+    require(uint256(initialAssetPrice) < type(uint128).max && initialAssetPrice < 0, "bad initial price"); // Be double sure the chainlink price isn't broken -uint128 is big enough starting price for any reasonable price feed.
+    epochInfo.lastEpochPrice = uint144(uint256(initialAssetPrice));
```

Comment:
Small improvement - check the price is above zero too on initialization.

And `epochInfo.lastEpochPrice` is a uint144 - so makes sense to cast it properly.

Reply:
Is it a typo? Should be require `initialAssetPrice > 0`.